db/seeds.rb

```diff
@@ -9,6 +9,11 @@
 #   end
 
 # users
+EncounterMonster.joins(:encounter).where(encounters: { user_name: "demo-one-encounter" }).destroy_all
+Encounter.where(user_name: "demo-one-encounter").destroy_all
+EncounterMonster.joins(:encounter).where(encounters: { user_name: "demo-many-encounter" }).destroy_all
+Encounter.where(user_name: "demo-many-encounter").destroy_all
```

Comment:
So, to make sure I understand what's going on here... this code makes it so that every time seed data is created for these users, the existing seed data is first deleted, ensuring that only the correct data appears each time seeding occurs. Yes? To avoid having multiple encounters on the user that's supposed to only have one, etc?

Reply:
I would have never thought to destroy specific seeds only! I usually burn everything down.